beacon_node/network/src/network_beacon_processor/sync_methods.rs

```diff
@@ -81,20 +81,21 @@ impl PeerGroupAction {
             AvailabilityCheckError::ReconstructColumnsError(_) => None, // internal error
             AvailabilityCheckError::KzgCommitmentMismatch { .. } => None, // should never happen after checking inclusion proof
             AvailabilityCheckError::Unexpected(_) => None,                // internal
-            AvailabilityCheckError::SszTypes(_) => None,                  // ??
-            AvailabilityCheckError::MissingBlobs => None, // TODO(das) internal for now
+            AvailabilityCheckError::MissingBlobs => {
+                Some(PeerGroupAction::block_peer(PeerAction::LowToleranceError))
```

Comment:
Missing blob doesn't feel as bad as serving invalid data, could possibly downgrade.

Reply:
Downgraded to HighToleranceError

Reply:
Yeah i think low is fine given we just penalise peer once in a single request now.